build/Dockerfile

```diff
@@ -6,8 +6,15 @@ RUN go build -o /nsx-ncp-operator/build/bin/nsx-ncp-operator ./cmd/manager
 
 FROM registry.access.redhat.com/ubi8/ubi:latest
 
+LABEL name="nsx-container-plugin-operator"
 LABEL maintainer="NSX Containers Team <nsx-container-dev@vmware.com>"
-LABEL description="A cluster operator to deploy nsx-ncp CNI plugin"
+LABEL summary="A cluster operator to deploy nsx-ncp CNI plugin"
+LABEL version="0.0.3"
```

Comment:
I saw in other places the version is 0.0.1, why here it's 0.0.3?

Reply:
Good point. That because I failed the first two verifications on the partner site, and the third one succeeded. I will align everything to 0.0.3

Reply:
Addresed! Now all references are to 0.0.3 version